web-app/js/exploreArea.js

```diff
@@ -446,10 +446,20 @@ function loadExploreArea(EYA_CONF) {
             } else if (state.speciesGroup != "ALL_SPECIES") {
                 fqParam = "&fq=species_group:" + state.speciesGroup;
             }
-            
+
+            // This is here in case tempalte string below doesn't work out
             var content = '<div class="infoWindow">Number of records: '+n.properties.count+'<br/>'+
                     '<a href="'+ EYA_CONF.contextPath +'/occurrences/search?q='+solrQuery+fqParam+
                     '&lat='+n.geometry.coordinates[1]+'&lon='+n.geometry.coordinates[0]+'&radius=0.05">View list of records</a></div>';
+
+            var content = `
+                <div class="infoWindow">
+                    Number of records: ${n.properties.count}
+                    <br />
+                    <a href="${EYA_CONF.contextPath}/occurrences/search?q=${solrQuery}${fqParam}&lat=${n.geometry.coordinates[1]}&lon=${n.geometry.coordinates[0]}&radius=0.05">
+                        View list of records
+                    </a>
+                </div>`;
```

Comment:
This should trigger variable redefinition error.

Also, template strings are ES6, so not available to us :/. Non-supporting browsers will stop execution at this point